ydb/library/yql/parser/pg_wrapper/arrow.h

```diff
@@ -1298,4 +1299,9 @@ TExecFunc FindExec(Oid oid);
 
 const NPg::TAggregateDesc& ResolveAggregation(const TString& name, NKikimr::NMiniKQL::TTupleType* tupleType, const std::vector<ui32>& argsColumns, NKikimr::NMiniKQL::TType* returnType);
 
+Numeric PgFloatToNumeric(double item, ui64 scale, int digits);
+
+template<typename T>
+std::shared_ptr<arrow::Array> PgConvertNumeric(const std::shared_ptr<arrow::Array>& value);
```

Comment:
this is kind of internal function, and the implementation of this template isn't visible, that may result into link error

please move implementation into _impl.h

PgFloatToNumeric is an internal function too

Reply:
Fixed